quickbuild/endpoints/tokens.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
+
+import xmltodict
+
+
+class Tokens:
+    """With tokens, one can authorize/unauthorize agents, or access agent
+    details including token value and latest usage information.
+    """
+
+    def __init__(self, quickbuild):
+        self.quickbuild = quickbuild
+
+    def authorize(self, agent_ip: str, agent_port: int = 8811) -> str:
+        """
+        Authorize a build agent to join the build grid.
+
+        Args:
+            agent_ip (str): The build agent IP address.
+            agent_port (int): The build agent port.
+
+        Returns:
+            str: identifier of the newly created token for the build agent
+        """
+        response = self.quickbuild._request(
+            'GET',
+            'tokens/authorize?ip={}&port={}'.format(agent_ip, agent_port)
```

Comment:
It's more correct to use params argument for request:

params=dict(ip=agent_ip, port=agent_port)